common/tools/dev-tool/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "noEmit": true,
+    "lib": ["ES6", "ESNext.AsyncIterable"],
```

Comment:
I thought we were trying out reference directives for asynciterable?

Reply:
I think that makes a lot of sense for the SDK packages to use reference directives, but for this command, I think just adding ESNext.AsyncIterable to the lib entry makes it more turn-key to manage.  We're not going to take a source dependency on `dev-tool` in any of the packages, so there aren't any of the risks associated with `lib` usage as in the SDK packages.

Reply:
yeah that's fair